src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -1329,9 +1331,11 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 	ISDBSubContext *ctx = codec_ctx;
 	const uint8_t *buf_pivot = buf;
 	int id = (*buf >> 2);
+#ifdef DEBUG // Fix [-Wunused-variable] when no debug
```

Comment:
Where is this DEBUG defined?

Reply:
I don't know:) It's not my definition. It was applied to log function earlier